corehq/apps/app_manager/suite_xml.py

```diff
@@ -1490,6 +1496,28 @@ def get_auto_select_assertions(self, case_id_xpath, mode, locale_arguments=None)
             )
         ]
 
+    def get_extra_case_id_datums(self, form):
+
+        def uses_usercase(form_):
+            actions = form_.active_actions()
+            if 'update_case' in actions and hasattr(actions['update_case'], 'update'):
+                return any_usercase_items(actions['update_case'].update.keys())
+            if 'case_preload' in actions:
+                return any_usercase_items(actions['case_preload'].preload.values())
+            return False
+
+        datums = []
+        if form.form_type == 'module_form' and uses_usercase(form) and self.is_usercase_enabled:
```

Comment:
if the form has 'user case' properties but `self.is_usercase_enabled` is false that seems like an error that should prevent the build.